doc/releases.md

```diff
@@ -283,3 +335,15 @@ This release includes a number of major improvements to the documentation and co
 - Exposed bokeh styling options for all plot types ([#134](https://github.com/pyviz/hvplot/pull/134))
 - Compatibility with latest HoloViews/GeoViews releases ([#113](https://github.com/pyviz/hvplot/pull/113), [#118](https://github.com/pyviz/hvplot/pull/118), [#134](https://github.com/pyviz/hvplot/pull/134))
 - Added control over tools ([#120](https://github.com/pyviz/hvplot/pull/120)) and legend position ([#119](https://github.com/pyviz/hvplot/pull/119))
+
+## Version 0.2.1
+
+**July 8, 2018**
+
+## Version 0.2.0
+
+**July 7, 2018**
+
+## Version 0.1.1
```

Comment:
I think this will look wrong when built. I propose one of three options:

1. Fill out some entries for these old releases
2. Comment out these entries for filling in later (if we think we will do that)
3. Collapse them into one sentence about the earliest releases, keeping only the dates.

Reply:
Ok I'll opt for 3.